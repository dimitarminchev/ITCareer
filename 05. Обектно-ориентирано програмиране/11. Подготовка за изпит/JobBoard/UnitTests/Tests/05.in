AddCategory Restaurants
AddCategory It
AddJobOffer Restaurants Waiter Romance 1000 onsite Sofia
GetAverageSalary It
End