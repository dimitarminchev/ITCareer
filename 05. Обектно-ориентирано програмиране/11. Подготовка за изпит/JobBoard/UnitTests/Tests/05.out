Created Category Restaurants!
Created Category It!
Created JobOffer Waiter in Category Restaurants!
The average salary is: 0.00 BGN